### ISSUE TITLE
Problems in training？

### ISSUE THREAD

**Issue by libenchong** [amaralibey/MixVPR#3]
Hi, Amar
Thanks for the great work.
When I train MixVPR，during 1.epoch，loss minimizes fastly，but during epoch2、3，loss doesn‘t fall，last epoch 4、5 loss maxmize fastly. Because the machine memory is too small, I set the batchsize to 20, and you set it to 120
![image](https://user-images.githubusercontent.com/37505417/218250629-6f641361-2862-493d-a394-24c7140e5030.png)
![image](https://user-images.githubusercontent.com/37505417/218250788-af8837ce-cec8-4c92-85b7-ac3dfec8c0d1.png)

**Comment by amaralibey** [comment]
Hi @libenchong,
I guess you used the same learning rate as the one I've used for batch size 120. I suggest you lower the learning rate to 0.01 and it should be okay.

**Comment by yayaYsmile** [comment]
How does MixVPR train? Error running main.py: Missing logger folder: LOGS/resnet50/lightning_logs
